app/src/androidTest/java/com/github/multimatum_team/multimatum/CalendarActivityTest.kt

```diff
@@ -33,9 +35,15 @@ class CalendarActivityTest {
 
     @Before
     fun setUp() {
+        Intents.init()
         hiltRule.inject()
     }
 
+    @After
+    fun teardown(){
```

Comment:
Nitpick (formatting): there should be a space between the parameter list and the {